modules/rostests/apitests/atl/CImage.cpp

```diff
@@ -264,25 +263,34 @@ START_TEST(CImage)
                                          aguidFileTypes,
                                          TEXT("All Image Files"), 0);
     ok(hr == S_OK, "Expected hr to be S_OK, was: %ld\n", hr);
-    ok(aguidFileTypes.GetSize() == 9, "Expected aguidFileTypes.GetSize() to be 8, was %d.", aguidFileTypes.GetSize());
-    ok(IsEqualGUID(aguidFileTypes[0], GUID_NULL), "Expected aguidFileTypes[0] to be GUID_NULL.\n");
-    ok(IsEqualGUID(aguidFileTypes[1], Gdiplus::ImageFormatBMP), "Expected aguidFileTypes[1] to be Gdiplus::ImageFormatBMP.\n");
-    ok(IsEqualGUID(aguidFileTypes[2], Gdiplus::ImageFormatJPEG), "Expected aguidFileTypes[2] to be Gdiplus::ImageFormatJPEG.\n");
-    ok(IsEqualGUID(aguidFileTypes[3], Gdiplus::ImageFormatGIF), "Expected aguidFileTypes[3] to be Gdiplus::ImageFormatGIF.\n");
-    ok(IsEqualGUID(aguidFileTypes[4], Gdiplus::ImageFormatEMF), "Expected aguidFileTypes[4] to be Gdiplus::ImageFormatEMF.\n");
-    ok(IsEqualGUID(aguidFileTypes[5], Gdiplus::ImageFormatWMF), "Expected aguidFileTypes[5] to be Gdiplus::ImageFormatWMF.\n");
-    ok(IsEqualGUID(aguidFileTypes[6], Gdiplus::ImageFormatTIFF), "Expected aguidFileTypes[6] to be Gdiplus::ImageFormatTIFF.\n");
-    ok(IsEqualGUID(aguidFileTypes[7], Gdiplus::ImageFormatPNG), "Expected aguidFileTypes[7] to be Gdiplus::ImageFormatPNG.\n");
-    ok(IsEqualGUID(aguidFileTypes[8], Gdiplus::ImageFormatIcon), "Expected aguidFileTypes[8] to be Gdiplus::ImageFormatIcon.\n");
+    ok(aguidFileTypes.GetSize() >= 9, "Expected aguidFileTypes.GetSize() to be >= 9, was %d.", aguidFileTypes.GetSize());
+    ok_int(IsEqualGUID(aguidFileTypes[0], GUID_NULL), TRUE);
+    ok_int(IsEqualGUID(aguidFileTypes[1], Gdiplus::ImageFormatBMP), TRUE);
+    ok_int(IsEqualGUID(aguidFileTypes[2], Gdiplus::ImageFormatJPEG), TRUE);
+    ok_int(IsEqualGUID(aguidFileTypes[3], Gdiplus::ImageFormatGIF), TRUE);
+    ok_int(IsEqualGUID(aguidFileTypes[4], Gdiplus::ImageFormatEMF), TRUE);
+    ok_int(IsEqualGUID(aguidFileTypes[5], Gdiplus::ImageFormatWMF), TRUE);
+    ok_int(IsEqualGUID(aguidFileTypes[6], Gdiplus::ImageFormatTIFF), TRUE);
+    ok_int(IsEqualGUID(aguidFileTypes[7], Gdiplus::ImageFormatPNG), TRUE);
+    ok_int(IsEqualGUID(aguidFileTypes[8], Gdiplus::ImageFormatIcon), TRUE);
+
+    static const char ImportFilter[] =
+        "All Image Files|*.BMP;*.DIB;*.RLE;*.JPG;*.JPEG;*.JPE;*.JFIF;*.GIF;*.EMF;*.WMF;*.TIF;*.TIFF;*.PNG;*.ICO|"
```

Comment:
Isn't "All Image Files" a localized string?

Reply:
`"All Image Files"` is a string literal specified above.

Reply:
Ah, you mean in the GetImporterFilterString call above? ok then this is nice - it allows per-app localization if needed :+1: